build.gradle

```diff
@@ -17,7 +17,7 @@
 
 apply plugin: 'scala'
 
-version = '0.6.3-2'
+version = '0.6.2.3'
```

Comment:
The master branch 'snappydata' already had this change. I don't understand why is it showing here in the diff.
In fact, I see all the three checkins already in this master branch. May be, this PR is not needed?